lib/yardstick/rule_description/classifier.rb

```diff
@@ -0,0 +1,105 @@
+module Yardstick
+  # Rule description composed of tokens which can be formatted
+  class RuleDescription
+    # Token classification
+    module Classifier
+      # List of classifiers
+      class List
+        extend Forwardable
+        include Concord.new(:classifiers), Enumerable
```

Comment:
Good catch.  I think this was how I originally used this class until I refactored and forgot to remove these.